src/universal-config.js

```diff
@@ -16,19 +16,19 @@ module.exports = {
           items: [
             {
               label: 'Vision',
-              to: '#',
+              to: 'https://libra.org/en-US/vision/',
```

Comment:
Remove all `en-US` and allow the header and `Accept-Language` to determine the path. If our i18n will always choose en-US I would still like to remove the explicit language.